website/content/docs/connect/ca/vault.mdx

```diff
@@ -29,6 +29,8 @@ must be met:
   were introduced in Vault 0.10.3. Prior versions of Vault are not
   compatible with Connect.
 
+~> **Note:** Do not use Vault v1.11.0+ as Consul's Connect CA provider — the intermediate CA will become unable to issue the leaf nodes required by service mesh, and by Consul client agents if using auto-encrypt or auto-config and using TLS for agent communication. If you are already using Vault 1.11+ as a Connect CA, refer to the [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/11308460105491) for more information about the underlying cause and recommended workaround.
```

Comment:
Can we change **the** to **this**?

Current: If you are already using Vault 1.11+ as a Connect CA, refer to **the** [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/11308460105491)

Change requested:
 If you are already using Vault 1.11+ as a Connect CA, refer to **this** [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/11308460105491)